src/mocks/handlers.ts

```diff
@@ -0,0 +1,36 @@
+import { ResponseComposition, rest, RestRequest } from 'msw'
+import { charmander, pickachu } from './data'
+
+export const handlers = [
+  rest.get(
```

Comment:
so the first 2 generics for this are the request body and response body, so for this endpoint it would look like:

```suggestion
  rest.get<undefined, ListCardsResponse>(
```

request body: `undefined` - this endpoint doesn't accept a request body
response body: `ListCardsResponse` - you already have a type for this!